docs/guides/tools-reference/file-transfer/comparing-data-transfer-utilities/index.md

```diff
@@ -0,0 +1,128 @@
+
+Different file types also involve varying metadata considerations. Object storage is self-contained, so it includes both data and metadata for easy search and access and retrieval. Structured file system storage also includes extensive metadata; when right-clicking a file in your desktop operating system, you can view its create/modify time, file type, version number, and more. Alternatively, block storage is highly performant but provides no metadata capabilities.
+
+### One-time Transfer vs. Recurring Syncs
+
+A specific utility may be well-suited for your use case depending on whether the data transfer is a single event or a recurring job. For example, an ad-hoc file transfer to a cloud server may only require a desktop-based data transfer utility, while a regularly scheduled backup job may require an automated tool or process that works at scale.
+
+## Comparing Data Transfer Utilities
+
+### Rsync
+
+[Rsync](https://rsync.samba.org/) is a standard Linux-based data transfer utility designed for syncing files between remote and local servers. Known for its flexibility, speed, and scriptable configuration, rsync is often a go-to utility for Linux administrators looking to quickly transfer and synchronize files between systems.
+
+#### Use Cases
+
+Rsync is ideal for syncing files between local and remote Linux machines in incremental backup and transfer use cases, as well as customizing large and complicated sync jobs with specific options and settings, like selecting a particular Linux shell to use in the transfer, or specifying files to analyze and exclude. Its delta-transfer algorithm reduces network traffic during syncs by only sending parts of a file that differ from the files on the recipient machine.
+
+#### Pros and Cons
+
+Rsync excels in use cases that require a proven method for transferring files and situations that call for extended scheduling and automation capabilities, such as using rsync with Linux cron jobs. However, it also requires familiarity with the command line, and it can be more system resource-intensive or slower as the number files copied increases.
+
+### Rclone
+
+[Rclone](https://rclone.org/) is a command-line utility for syncing files and directories to and from different cloud storage providers, servers, and workstations. As its name implies, the utility shares many similarities with rsync, including the same Linux commands (e.g., cp, mv, mount, ls). However, rclone was specially designed for rudimentary file transfers between cloud servers and on-premises servers and workstations, whereas rsync can be configured for more sophisticated file synchronization capabilities.
```

Comment:
This says that rclone shares commands with rsync--but I don't think rsync has commands? It has options you can specify, but that's not really the same syntax as a command/subcommand